packages/client/modules/email/components/SummaryEmail/MeetingSummaryEmail/WholeMeetingSummaryResult.tsx

```diff
@@ -30,8 +30,10 @@ const textStyle = {
   color: PALETTE.SLATE_700,
   fontFamily: FONT_FAMILY.SANS_SERIF,
   padding: '0px 48px 8px 48px',
-  fontSize: 14
-}
+  fontSize: 14,
+  whiteSpace: 'pre-line',
+  textAlign: 'left'
```

Comment:
-1 this could work for standups, but I don't think the Retro summary should be left aligned. 

Here's the left-aligned summary:

![Screenshot 2023-07-28 at 11 36 06](https://github.com/ParabolInc/parabol/assets/39854876/5c5b66a2-0b3c-4efe-9ca1-0c9384ded568)

And here's the summary without that change:
![Screenshot 2023-07-28 at 11 36 17](https://github.com/ParabolInc/parabol/assets/39854876/2d57acec-605a-4b87-805f-6ee31a0ec082)

Reply:
I'm not sure I agree with this - center-alignment is fine for 2, _maybe_ 3 lines, but any more than that and it gets hard to read. Looking at our internal retros over the past couple months, most of our retro summaries are 3-4 lines. Examples:
* 🔒 https://action.parabol.co/new-summary/nCI2Ten1hm
* 🔒 https://action.parabol.co/new-summary/k0TSLF3pkI
* 🔒 https://action.parabol.co/new-summary/nnSHTZuMPC
* 🔒 https://action.parabol.co/new-summary/oh12yvDQ2Y

@enriquesanchez @acressall what do y'all think?

Reply:
I'm not sure I agree with this - center-alignment is fine for 2, _maybe_ 3 lines, but any more than that and it gets hard to read. Looking at our internal retros over the past couple months, most of our retro summaries are 3-4 lines. Examples:
* 🔒 https://action.parabol.co/new-summary/nCI2Ten1hm
* 🔒 https://action.parabol.co/new-summary/k0TSLF3pkI
* 🔒 https://action.parabol.co/new-summary/nnSHTZuMPC
* 🔒 https://action.parabol.co/new-summary/oh12yvDQ2Y

@enriquesanchez @acressall what do y'all think?

Reply:
I agree that for longer texts left alignment is preferred for readability. We seem to be using both alignments in the same summary email (meeting summary is center aligned, topic summary is left aligned) and should stick to only one.